test-packages/integration-tests/test/auth-flows/auth-flow-util.ts

```diff
@@ -0,0 +1,55 @@
+import { resolve } from 'path';
+import { existsSync, readFileSync } from 'fs';
+import { createLogger } from '@sap-cloud-sdk/util';
+
+const logger = createLogger('oauth-util');
+
+export function loadLocalVcap(fileName = 'vcap-services.json') {
+  const fileContent = readFile(fileName);
+
+  process.env.VCAP_SERVICES = JSON.stringify(
+    JSON.parse(fileContent).VCAP_SERVICES
+  );
+  logger.info(`VCAP_SERVICES: ${process.env.VCAP_SERVICES}`);
+}
+
+export function readUserAccessToken(
+  fileName = 'user-access-token.json'
+): UserAccessTokens {
+  const fileContent = readFile(fileName);
+  return JSON.parse(fileContent);
+}
+
+export function readSysmtes(fileName = 'systems.json'): Systems {
+  const fileContent = readFile(fileName);
+  return JSON.parse(fileContent);
+}
```

Comment:
The implementations of those two (and also the implementation in `loadLocalVcap`) are duplicate, maybe create another `readJsonFile` function and use this one.